molo/forms/tests/test_admin.py

```diff
@@ -395,13 +398,20 @@ def test_convert_to_article(self):
         self.assertNotContains(response, 'Convert to Article')
 
     def test_export_submission_standard_form(self):
-        molo_form_page, molo_form_field = \
-            self.create_molo_form_page(parent=self.section_index)
-
+        molo_form_page = create_molo_form_page(
+            self.section_index,
+            title='test form',
+            display_form_directly=True,
+            save_article_object=False,
+            allow_anonymous_submissions=True,
+        )
+        molo_form_field = create_molo_form_formfield(
+            molo_form_page, 'singleline')
         self.client.force_login(self.user)
         answer = 'PYTHON'
-        response = self.client.post(molo_form_page.url, {
-            molo_form_field.label.lower().replace(' ', '-'): answer
+        response = self.client.post(molo_form_page.get_full_url(), data={
+            molo_form_field.clean_name: answer,
+            '': "abc"
```

Comment:
You can remove this entry from the dict now, right?

Reply:
removing